libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -656,7 +677,9 @@ void AP_MotorsMulticopter::output_logic()
 
         // set and increment ramp variables
         _spin_up_ratio = 1.0f;
-        _throttle_thrust_max -= spool_step;
+        const float spool_time = _spool_down_time > minimum_spool_time ? _spool_down_time : _spool_up_time;
+        const float spool_adjustment = -1.0f / (spool_time * _dt);
```

Comment:
why is this equation different. Shouldn't it be:
spool_step = _dt / _spool_up_time;

Reply:
Victim of a bad rebase, fixed it.